pallets/xcm-transactor/src/lib.rs

```diff
@@ -632,40 +730,110 @@ pub mod pallet {
 			});
 			Ok(())
 		}
-	}
 
-	impl<T: Config> Pallet<T> {
-		fn transact_in_dest_chain_asset_non_signed(
-			dest: MultiLocation,
-			fee_payer: T::AccountId,
-			fee_location: MultiLocation,
-			call: Vec<u8>,
-			origin_kind: OriginKind,
-
-			fee_amount: Option<u128>,
+		/// Manage HRMP operations
+		#[pallet::weight(0)]
+		pub fn hrmp_manange(
```

Comment:
```suggestion
		pub fn hrmp_manage(
```
and everywhere called